### DISCUSSION TITLE
Please remove me from types/roslib

### DISCUSSION THREAD

**Discussion by VorpalBlade** [DefinitelyTyped/DefinitelyTyped#57547] [Issues with a @types package] [answered]
Hi,

I no longer work with typescript or javascript at all. Please remove me from maintainership for types/roslib so I stop getting spammed with review requests.

I don't know if this is the proper way to request removal, but due to occupational burnout I don't have the energy to figure that out right now.

Thanks,
Arvid

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment, marked as answer]
Done #57556

**Comment by VorpalBlade** [comment]
Thanks!
